charts/kubescape-cloud-operator/values.yaml

```diff
@@ -256,6 +256,8 @@ operator:
   # Additional volumeMounts to be mounted on the websocket
   volumeMounts: []
 
+  TriggerSecurityFramework: true
```

Comment:
When will it ever be false?
Also, notice that all of the keys begin with lowercase and not uppercase:
`TriggerSecurityFramework` -> `triggerSecurityFramework`

Reply:
@dwertent 
(merged before your commented, opened a [new PR](https://github.com/kubescape/helm-charts/pull/229) with the small letter fix)

On helm installation, it should always be true.